src/test/unittest/unittest.sh

```diff
@@ -963,6 +963,20 @@ function check_pools() {
 	fi
 }
 
+#
+# create_obj_pool -- create an obj pool using pmempool
+#
+# This function expects 3 additional parameters. They are in order:
+# 1 - the pool file to be created
+# 2 - the size of the pool
+# 3 - the layout name
+#
+function create_obj_pool()
```

Comment:
fold this into the test, we do this manually everywhere else.
`expect_normal_exit $PMEMPOOL$EXESUFFIX create obj $POOL -s $SIZE`